DiscordBot.DataAccess/SheetsColumn.cs

```diff
@@ -0,0 +1,16 @@
+﻿using System;
+
+namespace DiscordBot.DataAccess
+{
+    internal readonly struct SheetsColumn
```

Comment:
I would use a class instead of a struct 🙂

Reply:
Thanks for that! Changed in 405d888 🙂